registry/storage/tagstore_test.go

```diff
@@ -210,24 +260,52 @@ func TestTagLookup(t *testing.T) {
 		t.Errorf("Lookup of descA returned %d tags, expected 2", len(tags))
 	}
 
-	tags, err = tagStore.Lookup(ctx, desc0)
+	tags, err = TagStore.Lookup(ctx, desc0)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	if len(tags) != 2 {
 		t.Errorf("Lookup of descB returned %d tags, expected 2", len(tags))
 	}
+	/// Should handle error looking up tag
```

Comment:
Needing to delimit test cases with comments is a sign that each case should be in its own (sub)test.